examples/inverse/plot_lcmv_beamformer_volume.py

```diff
@@ -83,15 +81,34 @@
 
 # You can save result in stc files with:
 # stc.save('lcmv-vol')
-
-clim = dict(kind='value', pos_lims=[0.3, 0.6, 0.9])
-stc.plot(src=forward['src'], subject='sample', subjects_dir=subjects_dir,
-         clim=clim)
+lims = [0.3, 0.6, 0.9]
+stc.plot(
+    src=forward['src'], subject='sample', subjects_dir=subjects_dir,
+    clim=dict(kind='value', pos_lims=lims), mode='stat_map',
+    initial_time=0.1, verbose=True)
 
 ###############################################################################
-# We can also visualize the activity on a "glass brain" (shown here with
-# absolute values):
+# Now let's plot this on a glass brain, which will automatically transform the
+# data to MNI Talairach space:
+
+# sphinx_gallery_thumbnail_number = 4
 
-clim = dict(kind='value', lims=[0.3, 0.6, 0.9])
-abs(stc).plot(src=forward['src'], subject='sample', subjects_dir=subjects_dir,
-              mode='glass_brain', clim=clim)
+stc.plot(
+    src=forward['src'], subject='sample', subjects_dir=subjects_dir,
+    mode='glass_brain', clim=dict(kind='value', lims=lims),
+    initial_time=0.1, verbose=True)
+
+###############################################################################
+# Finally let's get another view, this time plotting again a ``'stat_map'``
+# style but using volumetric morphing to get data to fsaverage space,
+# which we can get by passing a :class:`mne.SourceMorph` as the ``src``
+# argument to `mne.VolSourceEstimate.plot`. To save a bit of speed when
+# applying the morph, we will crop the STC:
+
+morph = mne.compute_source_morph(
+    forward['src'], 'sample', 'fsaverage', subjects_dir=subjects_dir,
+    zooms=7, verbose=True)
+stc.copy().crop(0.05, 0.18).plot(
+    src=morph, subject='fsaverage', subjects_dir=subjects_dir,
+    mode='stat_map', clim=dict(kind='value', pos_lims=lims),
+    initial_time=0.1, verbose=True)
```

Comment:
why is this so slow compared to the other two plots? Does the morph do some computation on demand? It's a bit hard to interact with the time series on this one, specially if you click on a time point very close to the vertical line (maybe because the samples are far apart?). 

That said, this PR is already a big improvement over what exists. So, feel free to merge it. I won't have time to look again the next 2 days.